ant-protocol/src/error.rs

```diff
@@ -69,6 +56,19 @@ pub enum Error {
     // The record already exists at this node
     #[error("The record already exists, so do not charge for it: {0:?}")]
     RecordExists(PrettyPrintRecordKey<'static>),
+
+    // ---------- Record Put errors
+    #[error("Error handling record put: {0}")]
+    PutRecordFailed(String),
+    #[error("Outdated record: with counter {counter}, expected any above {expected}")]
+    OutdatedRecordCounter { counter: u64, expected: u64 },
+
+    // Dev Note: add new variants above this one for backward compatibility with older protocol versions
```

Comment:
> above this one for backward compatibility

but still need to be AFTER the last valid one, i.e. OutdatedRecordCounter in this case ?